### PR TITLE
Update surfacehub-csp.md

### PR DESCRIPTION
**Pull request by DanPandre**
Document new ExchangeModernAuthEnabled setting

### CODE REVIEW

File: windows/client-management/mdm/surfacehub-csp.md [hunk 1]
```diff
@@ -143,6 +143,11 @@ The following diagram shows the SurfaceHub CSP management objects in tree format
 <p style="margin-left: 20px">Exchange server of the device account. Normally, the device will try to auto-discover the Exchange server. This field is only required if auto-discovery fails.
 
 <p style="margin-left: 20px">The data type is string. Supported operation is Get and Replace.
+ 
+ <a href="" id="deviceaccount-exchangemodernauthenabled"></a>**DeviceAccount/ExchangeModernAuthEnabled**
```
Comment (illfated) [inline windows/client-management/mdm/surfacehub-csp.md:147]: From what I can tell by looking at the other parts of the document, the indentation should follow the previous (and/or the following) lines. In this case, the document does not seem to suggest any indentation for this line.
```suggestion
<a href="" id="deviceaccount-exchangemodernauthenabled"></a>**DeviceAccount/ExchangeModernAuthEnabled**
```
(also removed whitespace from the end of the line)

**Comment by illfated** [comment]
Please note that I have made a direct PR to the file in your own PR branch, https://github.com/DanPandre/windows-itpro-docs/pull/1

If you find the whitespace clean-up acceptable, please merge the PR to add the changes to your own PR. 🙂

**Review by illfated — APPROVED** [review]
Regular, normal source file view:
- https://github.com/MicrosoftDocs/windows-itpro-docs/pull/9082/files

**Hide whitespace changes** (skips the inline whitespace corrections):
- https://github.com/MicrosoftDocs/windows-itpro-docs/pull/9082/files?diff=unified&w=1

**Rich Diff view** (condensed to only show the actual readable changes in rich text):
- https://github.com/MicrosoftDocs/windows-itpro-docs/pull/9082/files?diff=unified&w=1&short_path=c8eb73a#diff-c8eb73a834d52e07d011aec148417f62322d4f38b8db6b49511553a8c3d0c13d

**Comment by illfated** [comment]
#sign-off

**Comment by DanPandre** [comment]
Anything else that needs to happen to publish this?

**Comment by traya1** [comment]
merged PR with DanPandre's approval.

**Comment by illfated** [comment]
Thanks. Good job.

**Pull request merged**